src/main/resources/assets/tfc/lang/zh_cn.lang

```diff
@@ -99,6 +99,8 @@ tfc.tooltip.barrel_fluid=含 %1$d mB 的 %2$s
 tfc.tooltip.barrel_fluid_amount=%d mB
 tfc.tooltip.barrel_item=含 %1$dx% 2$s
 tfc.tooltip.barrel_item_in_fluid= 和 %1$dx %2$s
+tfc.tooltip.powderkeg_amount=含 %1$dx% 2$s
```

Comment:
why split %2 to % and 2

Reply:
Not sure what you're saying? It is exactly the same text as the Barrel. If it's wrong, let's fix it everywhere.

Reply:
OK I know it's my mistake, thank you for reminding me